examples/flask/pyproject.toml

```diff
@@ -6,7 +6,6 @@ authors = ["honeycombio"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-# honeycomb-beeline = "^2.17.2"
-honeycomb-beeline = { path = "../../../beeline-python" }
+# honeycomb-beeline = "^2.17.2" # uncomment if not using local beeline
```

Comment:
I am not sure actually if it's using the local beeline or if it's using the published beeline...